### ISSUE TITLE
Taxes.java:34-40: Implement tax payment to remote nodes....

### ISSUE THREAD

**Issue by 0pdd** [zold-io/java-api#61]
The puzzle `40-432fb579` from #40 has to be resolved:

https://github.com/zold-io/java-api/blob/4a0b4b4bed8a2e35dacb4def1b60fab6354cd1f7/src/main/java/io/zold/api/Taxes.java#L34-L40

The puzzle was created by Victor Noël on 28-Jul-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@0b295f2e-a16b-4cb4-975d-bbb72a45e7a2, thread: pool-57-thread-1
com.zerocracy.radars.github.ThrottledComments[74] java.lang.IllegalStateException: Can't post an identical message to zold-io/java-api#61: @llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue
```

[0.29.6](https://github.com/zerocracy/farm): Issue: zold-io/java-api#61, Action: `opened`

**Comment by 0crat** [comment]
Job [#61](https://github.com/zold-io/java-api/issues/61) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#61](https://github.com/zold-io/java-api/issues/61) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CAZUREFND/2191940415); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
Job `gh:zold-io/java-api#61` already assigned to @paulodamaso, can't assign to @vatavuk

**Comment by paulodamaso** [comment]
@0crat wait for #72 review

**Comment by 0crat** [comment]
> @0crat wait for #72 review ([here](https://github.com/zold-io/java-api/issues/61#issuecomment-412655826))

@paulodamaso The impediment for [#61](https://github.com/zold-io/java-api/issues/61) was registered successfully by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0pdd** [comment]
The puzzle `40-432fb579` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#81](https://github.com/zold-io/java-api/issues/81), [#82](https://github.com/zold-io/java-api/issues/82), [#83](https://github.com/zold-io/java-api/issues/83), [#84](https://github.com/zold-io/java-api/issues/84) are still not solved.

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by paulodamaso** [comment]
@llorllale No payment for me too

**Comment by llorllale** [comment]
@0crat pay @ypshenychka 8min

**Comment by llorllale** [comment]
@0crat pay @paulodamaso 35min

**Comment by llorllale** [comment]
@0crat pay 8min @ypshenychka

**Comment by llorllale** [comment]
@0crat pay 35min @paulodamaso

**Comment by 0crat** [comment]
> @0crat pay 8min @ypshenychka  ([here](https://github.com/zold-io/java-api/issues/61#issuecomment-444720097))

@llorllale The project is under-funded, you can't do it now, see [§49](http://www.zerocracy.com/policy.html#49)

**Comment by 0crat** [comment]
> @0crat pay 35min @paulodamaso  ([here](https://github.com/zold-io/java-api/issues/61#issuecomment-444720155))

@llorllale The project is under-funded, you can't do it now, see [§49](http://www.zerocracy.com/policy.html#49)

**Comment by paulodamaso** [comment]
@llorllale ping

**Comment by llorllale** [comment]
@paulodamaso not much I can do here..

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#116](https://github.com/zold-io/java-api/issues/116), [#117](https://github.com/zold-io/java-api/issues/117), [#82](https://github.com/zold-io/java-api/issues/82), [#84](https://github.com/zold-io/java-api/issues/84) are still not solved; solved: [#81](https://github.com/zold-io/java-api/issues/81), [#83](https://github.com/zold-io/java-api/issues/83).
